### PR TITLE
Adds qlog_dir documentation 

### PR DESCRIPTION
**Pull request by scw00**

### CODE REVIEW

File: doc/admin-guide/files/records.config.en.rst [hunk 1]
```diff
@@ -3884,6 +3884,12 @@ QUIC Configuration
 All configurations for QUIC are still experimental and may be changed or
 removed in the future without prior notice.
 
+.. ts:cv:: CONFIG proxy.config.quic.qlog_dir STRING nullptr
```
Comment (maskit) [inline doc/admin-guide/files/records.config.en.rst:3887]: It seems like we use `NULL` on the documentation. Maybe `nullptr` is too C++ specific.

Comment (scw00) [reply doc/admin-guide/files/records.config.en.rst]: fixed

File: doc/admin-guide/files/records.config.en.rst [hunk 2]
```diff
@@ -3884,6 +3884,12 @@ QUIC Configuration
 All configurations for QUIC are still experimental and may be changed or
 removed in the future without prior notice.
 
+.. ts:cv:: CONFIG proxy.config.quic.qlog_dir STRING NULL
```
Comment (randall) [inline doc/admin-guide/files/records.config.en.rst:3887]: There's a trailing whitespace here

**Comment by maskit** [comment]
[approve ci clang-format]

**Pull request merged**